documentation/reference/Overview_sv_SE.md

```
@@ -110,30 +110,10 @@ När en ny användare skapas skickar systemet normalt ett aktiveringsmail med en
-Riskregistret listar alla identifierade risker som kan påverka systemet. Det bör vara så omfattande som möjligt och inkludera alla identifierade risker samt en uppskattad sannolikhet att respektive risk inträffar, hur risken kan påverka om den inträffar, när den kan inträffa och hur ofta. Riskhantering används för att minimera sannolikheten att risken inträffar. 
-
-#### Bevarandehändelser
-
-En bevarandehändelse samlar metadata om händelser, specificerar och dokumenterar vilka objekt som har påverkats och vilken person eller mjukvara som varit inblandad. Dokumentation av händelser som modifierar ett objekt är avgörande för att upprätthålla digital härkomst, en nyckelfaktor för äkthet. Händelser som skapar nya relationer eller ändrar befintliga relationer är viktiga för att förklara dessa relationer. Även händelser som inte förändrar någonting, såsom giltighet och integritetskontroller av objekt, kan vara viktiga att registrera för förvaltningsändamål.
-
-#### Bevarandeaktör
-
-Enligt PREMIS (Preservation Metadata Implementation Strategies) är bevarandeaktörer (Preservation Agents) personer eller system som ansvarar för att utföra uppgifter eller roller relaterade till bevarandet av digitala objekt. Bevarandeaktörer kan omfatta både mänskliga aktörer (t.ex. arkivarier, kuratorer eller bevarandeadministratörer) och mjukvaruverktyg eller system (t.ex. innehållshanteringssystem, digitala bevarandesystem eller migrationsverktyg).
-
-I bevarandemetadata kopplas bevarandeaktörer ofta till specifika händelser eller åtgärder som sker under ett digitalt objekts livscykel. Genom att dokumentera dessa aktörer och deras roller kan organisationer upprätthålla en tydlig dokumentation över vem eller vad som har utfört olika bevarandeåtgärder och därigenom bidra till att säkerställa digitala resursernas långsiktiga tillgänglighet och integritet.
+Läs mer om gallring under administrationsguiden "Gallringspolicyer."
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)